Tests/Set-ADOPSPipelineRetentionSettings.Tests.ps1

```diff
@@ -0,0 +1,110 @@
+param(
+    $PSM1 = "$PSScriptRoot\..\Source\ADOPS.psm1"
+)
+
+BeforeAll {
+    Remove-Module ADOPS -Force -ErrorAction SilentlyContinue
+    Import-Module $PSM1 -Force
+}
+
+Describe 'Set-ADOPSPipelineRetentionSettings' {
+    Context 'Parameters' {
+        $TestCases = @(
+            @{
+                Name      = 'Organization'
+                Mandatory = $false
+                Type      = 'string'
+            },
+            @{
+                Name      = 'Project'
+                Mandatory = $true
+                Type      = 'string'
+            },
+            @{
+                Name      = 'Values'
+                Mandatory = $true
+                Type      = 'object'
+            }
+        )
+
+        It 'Should have parameter <_.Name>' -TestCases $TestCases {
+            Get-Command Set-ADOPSPipelineRetentionSettings | Should -HaveParameter $_.Name -Mandatory:$_.Mandatory -Type $_.Type
+        }
+    }
+
+    Context 'Getting Pipeline Retention Settings' {
+        BeforeAll {
+            $OrganizationName = 'DummyOrg'
+            $Project = 'DummyProject'
+            $Setting = 'artifactsRetention'
+
+#             Mock -CommandName InvokeADOPSRestMethod -ModuleName ADOPS -MockWith {
```

Comment:
If this mock isn't needed, please remove. It does seem to be needed though.. And if so, uncomment 😉